### ISSUE TITLE
Healthy Events not being emiited

### ISSUE THREAD

**Issue by blazedd** [neumino/rethinkdbdash#183]
I'm currently testing an app which needs to know it's it's connection status to our cluster and we're having a difficult time triggering the `healthy` event from triggering on the poolMaster. We are using iptables to drop either outbound or inbound traffic.

When the application starts without a connection it seems the `healthy` event never triggers. We do get the `Entering Slow Mode` and `Exiting Slow Mode` messages, but the event for the health checks rarely if at all occurs.

I feel like I'm missing something pretty essential as the entire pool has no active connection. Queries are still queued and handled if we drop the iptables rules.

**Comment by neumino** [comment]
Can you a script where you listen for the event? And the iptables used?

I haven't played with iptables for some time, but if I'm not mistaken, with iptables you can break a connection without getting an event, in which case the connection dies only when it's pinged.

**Comment by neumino** [comment]
Slow growth doesn't mean that no connection are available. It means that we won't try to open tons of connections at once because we failed to open once.
The other connections are still there.

**Comment by blazedd** [comment]
Sure thing! These firewall rules will drop all traffic in the same manor that a server crashing or losing connection in the wild would. No hangup or rejection - just dropping all packets.

*IPTables Rules* 

```
iptables -A OUTPUT -d <remote-server-ip> -j DROP
iptables -A INPUT -s <remote-server-ip> -j DROP
```

*Javascript* - Nothing overly fancy here

```javascript
app.r = require('rethinkdbdash')(config);
var master = app.r.getPoolMaster();
master.on('healthy', function(healthy){
	if(healthy){
		app.info('Reconnected to RethinkDB pool');
	}else{
		app.warn('Lost connection with RethinkDB pool');
	}
});
```

*Config*

```javascipt
{
	discovery: false,
	pool: true,
	buffer: 50,
	max: 1000,
	timeout: 5,
	timeoutError: 1000,
	silent: true,
	servers: [...],
	db: '...'
```

**Comment by neumino** [comment]
TL;DR: Linux needs time to find that a connection is not available if you just block it with iptables. While it's not optimal, it's working as intended.

More information here:
https://github.com/rethinkdb/rethinkdb/issues/4572 and https://github.com/rethinkdb/rethinkdb/issues/4641
